worker/main.go

```diff
@@ -179,6 +207,27 @@ outerloop:
 	//TODO send to storage
 	fmt.Println(buf.String())
 
+	notifyMasterOnProcessCompletion(graph.Id)
+}
+
+func notifyMasterOnProcessCompletion(graphId uuid.UUID) {
+	requestOptions := grequests.RequestOptions{
+		JSON: struct {
+			RequestId  uuid.UUID
+			InstanceId string
+		}{
+			RequestId:  graphId,
+			InstanceId: conf.Own.Instanceid,
+		},
+		Headers: map[string]string{"Content-Type": "application/json", "X-Auth": conf.ApiKey},
+	}
+
+	resp, err := grequests.Post(getMasterURL()+"/worker/done", &requestOptions)
```

Comment:
Zo te zien wordt resp niet gebruikt, dan zou je gewoon `_, err := ....` kunnen doen, dan is de `defer resp.Close()` niet nodig.